build/scripts/build-utils.ps1

```diff
@@ -353,6 +359,19 @@ function Get-MSBuildDir([switch]$xcopy = $false) {
     return $both[1]
 }
 
+
+# Dose this version of Visual Studio meet our minimum requirements for building.
+function Test-SupportedVisualStudioVersion([string]$version) { 
+    if (-not ($version -match "^([\d.]+)(\+|-)?.*$")) { 
```

Comment:
Why not just call Version.TryParse?

Reply:
Because I need to strip off the prelease modifiers here.

Reply:
@jaredpar Oh, that's what that regex does? I then find your lack of comments disturbing.

Reply:
Comment added